pyomo/contrib/parmest/examples/reactor_design/updatesuffix_example.py

```diff
@@ -0,0 +1,67 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2025
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+from pyomo.common.dependencies import numpy as np, pandas as pd
+from os.path import join, abspath, dirname
+import pyomo.contrib.parmest.parmest as parmest
+from pyomo.contrib.parmest.examples.reactor_design.reactor_design import (
+    ReactorDesignExperiment,
+)
+
+import pyomo.environ as pyo
+from pyomo.contrib.parmest.utils.model_utils import update_model_from_suffix
+
+
+def main():
```

Comment:
One example driver is called `main`, and the other one is called `run_reactor_update_suffix_items`. It's inconsistent. (Similar to other comment, not a deal-breaker, just a bit odd.)

Reply:
I have no strong preference - pick whichever one brings you joy.